src/components/ManagersTable/ManagersTable.tsx

```diff
@@ -64,6 +68,18 @@ export const ManagersTable = (): JSX.Element => {
         filterRows(email, searchValue),
     );
 
+  useEffect(() => {
+    if (id && baseBreadcrumbs && baseBreadcrumbs.length > 0) {
+      const managersCrumb: Breadcrumb[] = [
+        {
+          icon: <Svg id="manager-outlined" width="15" height="15" />,
+          label: t('managers'),
+        },
+      ];
+      dispatch(breadcrumbs.actions.setBreadcrumbs([...baseBreadcrumbs, ...managersCrumb]));
```

Comment:
y can simplify this dispatch(breadcrumbs.actions.setBreadcrumbs([...baseBreadcrumbs, {
icon,
label
}]));

Reply:
done